src/main/python/util.py

```diff
@@ -52,13 +52,15 @@ def is_rawhid(dev):
     return dev["interface_number"] == 1
 
 
-def find_vial_devices(sideload_vid=None, sideload_pid=None):
+def find_vial_devices(via_stack_json, sideload_vid=None, sideload_pid=None):
     from vial_device import VialBootloader, VialKeyboard
 
     filtered = []
     for dev in hid.enumerate():
         if dev["vendor_id"] == sideload_vid and dev["product_id"] == sideload_pid and is_rawhid(dev):
             filtered.append(VialKeyboard(dev, sideload=True))
+        elif str(dev["vendor_id"] * 65536 + dev["product_id"]) in via_stack_json["definitions"] and is_rawhid(dev):
```

Comment:
I think this should be lower priority than vial-enabled keyboard, otherwise VIA definitions will override vial firmware json.
i.e. move it below the next condition

Reply:
fixed